view/adminhtml/templates/info/billet.phtml

```diff
@@ -9,10 +9,35 @@
  * @link        http://www.mundipagg.com
  */
 ?>
+<?php
+
+/* @var MundiPagg\MundiPagg\Block\Payment\Info\Billet $this */
+$this;
```

Comment:
Esse $this fica solto assim mesmo??

Reply:
Sim, para ser referenciado no DocBlock, por sem uma view não tem referência. Igual um Controller e sua view nos frameworks